packages/api/src/utility.ts

```diff
@@ -49,7 +49,7 @@ export function mapSubscriptionsAsCsv(subscriptions: Subscription[]) {
       'paymentMethodID',
       'deactivationDate',
       'deactivationReason'
-    ].join(',') + '\n'
+    ].join(';') + '\n'
```

Comment:
Sometimes, users use commas in their fields such as the address (e.g. example street, 13). This causes wrong column separation within the csv file.

Reply:
Shouldn't we be RFC compliant https://www.ietf.org/rfc/rfc4180.txt? Since CSV stands for: Comma-Separated Values Escape commas?

Reply:
Good point. Alternatively, I can sanitize all user input files. What do you think? Would that be a better solution? Probably, right?

Reply:
Ok. Refactored that.